heliopy/data/util.py

```diff
@@ -161,6 +161,18 @@ def units_attach(data, units):
     return timeseries_data
 
 
+def cdf_units(cdf_):
+    units = collections.OrderedDict()
+    for variable_name in list(cdf_.keys()):
+        if 'UNITS' in cdf_[variable_name].attrs:
+            try:
+                temp_ = u.Unit(cdf_[variable_name].attrs['UNITS'])
+                units[variable_name] = temp_
+            except ValueError:
+                units[variable_name] = u.dimensionless_unscaled
```

Comment:
Currently, some CDF files have different ways to manage their units. Like, cm^-3 is written as &/cc, which means the same thing, since cubic centimetres, but the problem is that AstroPy units won't recognize the u.Unit('&/cc'). So, I'm thinking of adding some manual entries for the same.